### ISSUE TITLE
Can't get good result on FD003 dataset

### ISSUE THREAD

**Issue by Lunarli** [ruibing-jin/Bi_LSTM_TS#1]
**I can get Comparable result on FD001、2、4dataset**，but when I run your code on the FD003 dataset，I have try many times but get worse result，as shown in the following figure（average value of the last 10 epoch）
![image](https://user-images.githubusercontent.com/49749330/233825477-db1b0f1e-d71d-4742-99f9-892156b7de4b.png)
Can you give me some advice how to solve it?

Besides，I'm confused about the value in this code  `x = x.split(1, 1)[0]`  taking **0** instead of **-1**
```python
x, hidden = self.bi_lstm1(x_in)
x_split = torch.split(x, (x.shape[2] // 2), 2)
x = x_split[0] + x_split[1]
x = self.drop1(x)
x, hidden = self.bi_lstm2(x)
x_split = torch.split(x, (x.shape[2] // 2), 2)
x = x_split[0] + x_split[1]
x = self.drop2(x)
x = x.split(1, 1)[0]
```

**Comment by ruibing-jin** [comment]
Hi,

For results in FD003, I have checked all my experimental results FD003 and all their results are similar to that reported in our paper. I guess thie issue in your experiments may be related to the systems environment, especially for the ubuntu version, cuda version, pytorch version and the GPU card type. This may affect your experimental results. 

For the code question x = x.split(1, 1)[0], since we use the bi-directional LSTM, both output from [0] and that from [-1] includes all temporal input data. So, in theory, you may get the similar performances, if you modify it as [-1] during the training process.

Hope these answers can help you. Best wishes for you.

**Comment by Lunarli** [comment]
Thank you for your answer ！I will double check it, also I would like to know **how you got the results in your paper**, from your code I found that 5 rounds of cross-validation were divided, but **the validation set was not used**

**Comment by ruibing-jin** [comment]
Hi,

The code released on Github is only for your reference, which is not consistent to our experiments' code. Some functions are removed. Actually, the function of using the validation set is reserved in this code.

If you want to use the validation set, you can modify line 207: self.dataset.reset('test') in the model.py as "self.dataset.reset('val')".

**Comment by Lunarli** [comment]
OK，I‘ll try it.
